src/Services/AppointmentService.php

```diff
@@ -189,6 +223,200 @@ public function search($search, $isAndCondition = true)
         return $processingResult;
     }
 
+    /**
+     * Decodes a UUID string into bytes.
+     *
+     * @param string $puuid The UUID string to decode.
+     * @return string|null Returns the decoded UUID as bytes, or null if decoding fails.
+     */
+    private function decodeUuid(string $puuid)
```

Comment:
if is possible add the return type:
private function decodeUuid(string $puuid): TypeToReturn